.github/workflows/docker-image.yml

```diff
@@ -0,0 +1,18 @@
+name: Docker Image CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
Don't specify the `master` branch when it's triggered on PR because some contributors don't use the master branch to submit their PRs.

Reply:
Will now be triggered on all of the branches.